examples/runqslower/src/bpf/runqslower.bpf.c

```diff
@@ -1,9 +1,15 @@
 // SPDX-License-Identifier: GPL-2.0
 // Copyright (c) 2019 Facebook
+
 #include "vmlinux.h"
 #include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
 #include "runqslower.h"
 
+struct task_struct_pre_5_14 {
+	volatile long int state;
```

Comment:
nit: volatile not really necessary, you are going to be bpf_probe_read()'ing it either way